src/components/multi-action-button/multi-action-button.spec.tsx

```diff
@@ -57,13 +59,14 @@ describe("MultiActionButton", () => {
     let mainButton: ReactWrapper;
 
     beforeEach(() => {
+      childOnClickCB = jest.fn();
       container = document.createElement("div");
       container.id = "enzymeContainer";
       document.body.appendChild(container);
       wrapper = mount(
         <MultiActionButton text="Main Button">
-          <Button>First</Button>
-          <Button>Second</Button>
+          <Button onClick={childOnClickCB}>First</Button>
+          <Button onClick={childOnClickCB}>Second</Button>
```

Comment:
I'm not sure the value of adding these `onClick` handlers no we're not actually asserting anything to do with them. I think this spec file can probably be left alone as we're no longer adding any tests to it (nor do we need to adjust anything to account for the fixed behaviour).